Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -156,16 +156,20 @@ var TouchableOpacity = React.createClass({
   },
 
   _opacityInactive: function(duration: number) {
-    var childStyle = flattenStyle(this.props.style) || {};
     this.setOpacityTo(
-      childStyle.opacity === undefined ? 1 : childStyle.opacity,
+      this._getChildStyleOpacityWithDefault(),
       duration
     );
   },
 
   _opacityFocused: function() {
     this.setOpacityTo(this.props.focusedOpacity);
   },
+  
+  _getChildStyleOpacityWithDefault: function() {
+   var childStyle = flattenStyle(this.props.style) || {};
+   return childStyle.opacity === undefined ? 1 : childStyle.opacity;
```

Comment:
I know it was like this before but could you change this to `== undefined` so it also matches `null`.